test/issue_fail.test.js

```diff
@@ -0,0 +1,44 @@
+const nock = require('nock')
+const myProbotApp = require('..')
+const { Probot } = require('probot')
+
+const payloadFail = require('./fixtures/payloadFail.json')
+
+// 실제 주소와 연결을 하지 않고 테스트 진행
+nock.disableNetConnect()
+
+describe('Start Pull Request Open', () => {
+  let probot
+
+  beforeEach(() => {
+    probot = new Probot({})
+
+    const app = probot.load(myProbotApp)
+    app.app = () => 'test'
+  })
+
+  test('when no markdown files are editted', async () => {
+    nock('https://api.github.com')
+      .post('/app/installations/1189219/access_tokens')
+      .reply(200, { token: 'test' })
+
+    nock('https://api.github.com')
+      .get('/repos/Im-D/probot-practice/pulls/32/files')
+      .reply(200, [
+        {
+          filename: 'test.js'
+        }
+      ])
+
+    nock('https://api.github.com')
+      .patch('/repos/Im-D/probot-practice/pulls/32', body => {
+        if (body.body === '') {
```

Comment:
```suggestion
        if (body.body === '') return true
```

위와 같이 한 줄로 끝내는게 개인적으로는 좀 더 깔끔할 것 같은데 어떠신가요?

Reply:
개인적으로 if문은 한줄이여도 일관성을 맞추면서 괄호를 넣는 것을 선호하는 편이라 이렇게 작업을 했는데, 이번에 rule을 정하는 것도 좋을 듯 합니다.